packages/commerce/tailwind.config.js

```diff
@@ -0,0 +1,48 @@
+const defaultTheme = require('tailwindcss/defaultTheme')
+const colors = require('tailwindcss/colors')
+
+module.exports = {
+  mode: 'jit',
+  darkMode: 'class',
+  purge: {
+    content: ['./src/**/*.tsx', './src/**/*.mdx'],
+  },
+  theme: {
+    extend: {
+      colors: {
+        ...colors,
+      },
+      screens: {
+        print: {raw: 'print'},
+      },
+      typography: (theme) => ({
+        DEFAULT: {
+          css: {
+            a: {
+              color: theme('colors.rose.600'),
```

Comment:
I cut and pasted this config but it feels like we could have a shared config where the theme colors are the only thing that really change for us.